portal/views/teacher/teach_new.py

```diff
@@ -489,6 +490,96 @@ def teacher_move_class_new(request, access_code):
 
 @login_required(login_url=reverse_lazy('login_new'))
 @user_passes_test(logged_in_as_teacher, login_url=reverse_lazy('login_new'))
+def teacher_move_students_new(request, access_code):
```

Comment:
This is because both the old and new methods are used in the urls.py file, so even though they are imported from two different files they need to have different names otherwise the url by default points to the old one.